### PR TITLE
PlanChooseInterface.js userprofile.js and brandprofile.js static pages made as responsive

### PR DESCRIPTION
**Pull request by BHS-Harish**
**Issue #89 fixed**

### CODE REVIEW

**Comment by RohitGupta1237** [comment]
@BHS-Harish  hey just one issue with this page alignment of brand profile and dp and spacing between heading 
Screenshot 2024-06-27 at 11 14 36 PM

**Comment by RohitGupta1237** [comment]
@BHS-Harish  in figma the dp is buttons all are aligned in one line here alignment is little odd 

Screenshot 2024-06-27 at 11 26 10 PM

IMG_0738
IMG_0739

**Comment by RohitGupta1237** [comment]
@BHS-Harish  hey can we connect ?

File: frontend/App.js [hunk 1]
```diff
@@ -39,8 +39,11 @@ import InfluencerDeleteAccountPage from "./screens/Influencer/Settings/Support/I
 import InfluencerContactUs from "./screens/Influencer/Settings/Support/InfluencerContactUs";
 import InfluencerHelpCenter from "./screens/Influencer/Settings/Support/InfluencerHelpCenter";
 import InfluencerManageAccount from "./screens/Influencer/Settings/InfluencerAccountManage";
+<<<<<<< HEAD
```
Comment (Kajol-Kumari) [inline frontend/App.js:42]: miss during conflict resolve

File: frontend/screens/BrandProfile.js [hunk 2]
```diff
@@ -67,162 +67,181 @@ const BrandProfile = ({ navigation }) => {
...
           console.error("Error fetching minimum requirements:", error)
         );
-        getBrandProfile(brandId, showAlert).then((data) => setBrand(data));
+      getBrandProfile(brandId, showAlert).then((data) => setBrand(data));
     }
   }, [brandId, token]);
 
 
   return (
+<<<<<<< HEAD
```
Comment (Kajol-Kumari) [inline frontend/screens/BrandProfile.js:76]: Same, miss during conflict resolve

**Pull request closed without merging**